Sources/Commandant/ArgumentType.swift

```diff
@@ -12,21 +12,21 @@ public protocol ArgumentType {
 	static var name: String { get }
 
 	/// Attempts to parse a value from the given command-line argument.
-	static func fromString(string: String) -> Self?
+	static func fromString(_ string: String) -> Self?
 }
 
 extension Int: ArgumentType {
 	public static let name = "integer"
 
-	public static func fromString(string: String) -> Int? {
+	public static func fromString(_ string: String) -> Int? {
 		return Int(string)
 	}
 }
 
 extension String: ArgumentType {
 	public static let name = "string"
 
-	public static func fromString(string: String) -> String? {
+	public static func fromString(_ string: String) -> String? {
```

Comment:
These should be renamed `from(string: String)` to be in the Swift 3 style. (With an `@available` that points to the rename.)